.github/workflows/main.yaml

```
@@ -17,31 +18,81 @@
     runs-on: ubuntu-latest
     outputs:
       min_coverage: ${{ steps.extract.outputs.min_coverage }}
+      coverage_excludes: ${{ steps.extract.outputs.coverage_excludes }}
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - id: extract
         name: Extract configuration values
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:6:27: Double quote to prevent globbing and word splitting

(shellcheck)